docs/source/reference/rest.md

```diff
@@ -136,7 +145,7 @@ Authorization header.
 ### Use requests
 
 Using the popular Python [requests](https://docs.python-requests.org)
-library, here's example code to make an API request for the users of a JupyterHub
+library, here's an example code to make an API request for the users of a JupyterHub
```

Comment:
I don't think the article here is correct. I would say "here is code" or "here is some code" not "here is a code"